### DISCUSSION TITLE
Nice!

### DISCUSSION THREAD

**Discussion by oenu** [colinlienard/gitlight#259] [General]
Extremely cool! Do you have a discord server for chatting about development? I am interested in contributing but dont want to step on others work

**Comment by colinlienard** [comment]
Thanks for your interest in the project @oenu!
I haven't created a discord server, contributions are made through issues or spontaneous PRs.
In fact, for me, the project is kind of "finished" in the sense that I don't plan to add any major new features, but if you have any ideas or things that you would like to implement, you can talk about them here!
